components/hab/src/cli.rs

```diff
@@ -905,6 +906,29 @@ fn sub_pkg_build() -> App<'static, 'static> {
     sub
 }
 
+fn sub_pkg_download() -> App<'static, 'static> {
+    let sub = clap_app!(@subcommand download =>
+    (about: "Download Habitat artifacts (including dependencies and keys) from Builder")
+    (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+    (@arg BLDR_URL: --url -u +takes_value {valid_url} default_value(habitat_core::url::DEFAULT_BLDR_URL)
+        "Specify an alternate Builder endpoint. If not specified, the value will \
+         be taken from the HAB_BLDR_URL environment variable if defined. (default: \
+         https://bldr.habitat.sh)")
+    (@arg CHANNEL: --channel -c +takes_value default_value[stable] env(ChannelIdent::ENVVAR)
+        "Install from the specified release channel (default: stable)")
+    (@arg DOWNLOAD_DIRECTORY: --download_directory +takes_value "The path to store downloaded artifacts")
```

Comment:
The underscore here is unlike any other Hab command where dash/hyphen is the norm.  Verifying in the UX Design spec, it should indeed be a hyphen 😃 

It can be remedied easily enough with:

`--("download-directory")`